host-interaction/hardware/memory/manipulate-linux-process-memory.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: manipulate Linux process memory
```

Comment:
the APIs referenced here are not linux specific (they're part of the c standard library) so lets loosen that restriction

Reply:
I wonder how useful this rule will be as one of these apis will be used in 99% of samples? I vote to remove this generic rule.